playground/tests/helpers.ts

```diff
@@ -1,7 +1,9 @@
+import fs from 'node:fs';
+import assert from 'node:assert';
 import { createStrapi } from '@strapi/strapi';
-import fs, { PathLike } from 'fs';
+import type { Core } from '@strapi/types';
 
-let instance;
+let instance: Core.Strapi | undefined;
```

Comment:
This is amazing. I was wondering why the types were not working in the test files.